dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/ChannelWrappedInvoker.java

```diff
@@ -71,7 +71,9 @@ protected Result doInvoke(Invocation invocation) throws Throwable {
                 currentClient.send(inv, getUrl().getMethodParameter(invocation.getMethodName(), SENT_KEY, false));
                 return AsyncRpcResult.newDefaultAsyncResult(invocation);
             } else {
-                CompletableFuture<AppResponse> appResponseFuture = currentClient.request(inv).thenApply(obj -> (AppResponse) obj);
+                final String methodName = RpcUtils.getMethodName(invocation);
+                final int timeout = calculateTimeout(invocation, methodName);
+                CompletableFuture<AppResponse> appResponseFuture = currentClient.request(inv, timeout, null).thenApply(obj -> (AppResponse) obj);
```

Comment:
I have two thoughts over this issue:
1. Should we implicitly set all server push invokers to one-way mode inside the framework? 
2. If 1 is not right, I think adding a timeout to server push RPC call is necessary, but simply using `url.getParameter("timeout")` would be enough, times spent during the should not be considered into the calculation.